homeassistant/helpers/entity_component.py

```diff
@@ -319,8 +347,16 @@ def async_add_entities(self, new_entities, update_before_add=False):
         if not new_entities:
             return
 
-        tasks = [self._async_process_entity(entity, update_before_add)
-                 for entity in new_entities]
+        @asyncio.coroutine
+        def async_process_entity(new_entity):
```

Comment:
It is only a cleanup. I don't know why I make that as member function. I cleanup my Fails while that is only needed on this please and make no sense on other please.